.github/workflows/stable-release.yml

```diff
@@ -0,0 +1,63 @@
+name: Stable Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      specific_version:
+        type: string
+        description: Specific version number (Optional). Default will be the current version plus 0.0.1.
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GHCR_TOKEN }}
+      - name: Set up Git
+        run: |
+          git config --global user.email "liugs963@gmail.com"
+          git config --global user.name "goldmedal"
```

Comment:
Using personal email and name is not a good idea.  May use `github` or `ci` will be better.